src/components/tabs/__tests__/Tabs.spec.tsx

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Button } from "../../button";
+import { Tab, Tabs } from "../Tabs";
+import {
+  ACTIVE_TAB_CLASSNAME,
+  INACTIVE_TAB_CLASSNAME,
+  NEXT_BUTTON_LABEL,
+  PREVIOUS_BUTTON_LABEL,
+  TAB1_CAPTION,
+  TAB1_CONTENT,
+  TAB1_LABEL,
+  TAB2_CAPTION,
+  TAB2_CONTENT,
+  TAB2_LABEL,
+  TAB3_CONTENT,
+  TAB3_LABEL,
+} from "../testConstants";
+
+function TabsTestComponent() {
+  const [currentStep, setCurrentStep] = React.useState(0);
+  return (
+    <Tabs step={currentStep} handleTab={setCurrentStep} wizard>
+      <Tab heading={TAB1_LABEL} caption={TAB1_CAPTION}>
+        {TAB1_CONTENT}
+      </Tab>
+      <Tab heading={TAB2_LABEL} caption={TAB2_CAPTION}>
+        {TAB2_CONTENT}
+      </Tab>
+      <Tab heading={TAB3_LABEL} disabled>
+        {TAB3_CONTENT}
+      </Tab>
+    </Tabs>
+  );
+}
+
+test("active styles are applied via classname on click", () => {
+  render(<TabsTestComponent />);
+
+  const firstTab = screen.getByText(TAB1_LABEL);
+  const secondTab = screen.getByText(TAB2_LABEL);
+  const firstTabContainer = firstTab.parentElement;
+  const secondTabContainer = secondTab.parentElement;
+
+  expect(firstTabContainer).toHaveClass(ACTIVE_TAB_CLASSNAME);
+  expect(firstTabContainer).not.toHaveClass(INACTIVE_TAB_CLASSNAME);
+
+  userEvent.click(secondTab);
+  expect(firstTabContainer).not.toHaveClass(ACTIVE_TAB_CLASSNAME);
+  expect(secondTabContainer).toHaveClass(ACTIVE_TAB_CLASSNAME);
+  expect(firstTabContainer).toHaveClass(INACTIVE_TAB_CLASSNAME);
+  expect(secondTabContainer).not.toHaveClass(INACTIVE_TAB_CLASSNAME);
```

Comment:
55f9c0ec13ea41076f3196cf6af7a1e7e7698b0c 1587684f8cf64cb5849d827d45e9866311f650a9 

I used `const ariaSelected = activeTab === index ? "true" : "false";` instead of `const ariaSelected = isActive ? "true" : "false";` since isActive is false on index = 0.

LMK if there's anything else, thanks :D